buildSrc/src/toolkits/gradle/changelog/ReleaseCreator.kt

```diff
@@ -6,7 +6,7 @@ package toolkits.gradle.changelog
 import java.io.File
 import java.time.LocalDate
 
-class ReleaseCreator(private val unreleasedFiles: List<File>, private val nextReleaseFile: File) {
+class ReleaseCreator(private val unreleasedFiles: Collection<File>, private val nextReleaseFile: File) {
```

Comment:
What does this change?

Reply:
The call site switch to returning a Set<File> instead, but since we dont care about the actual list vs set, I went up a level in the inheritance tree